lib/components/narrative/trip-tools.js

```diff
@@ -169,7 +171,20 @@ LinkButton.propTypes = {
   url: PropTypes.string
 }
 
+const makeIcon = memoize((SvgIcon, iconType) => {
+  const IconComponent = (props) => {
+    return (
+      <StyledIconBase>
+        <SvgIcon iconName={iconType} {...props} />
```

Comment:
Is this the approach we want to take? This would require shipping every icon we want in the `config.js`. Could we not use dynamic imports instead to import directly from fa?

Reply:
Shipping every icon in config makes sense, as the label for the popup button is defined in config.

Reply:
But if you wanted to use a `fa` import wouldn't you have to define that/import it during the build step in the config instead of just importing it dynamically?

Reply:
A dynamic import assumes we have a predetermined set of known icons like the ones from `SvgIcon`. Do we want to do that?

Reply:
Also, defining an icon for the popup link/button remains optional, the default is still the "external link" icon.

Reply:
I just don't like having to define all the icons as svg in the config and the pulling them out. Would prefer something less cumbersome

Reply:
Turns out that, from the config file, you can import any icon from any applicable package referenced by `package.json`, so extra SVG files are not required in these cases.

Reply:
And no dynamic imports are required that way.